packages/fxa-settings/src/pages/Signup/index.tsx

```diff
@@ -0,0 +1,490 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React, { useState } from 'react';
+import { RouteComponentProps } from '@reach/router';
+import { useForm } from 'react-hook-form';
+import { logViewEvent, usePageViewEvent } from '../../lib/metrics';
+import { FtlMsg } from 'fxa-react/lib/utils';
+import FormPasswordWithBalloons from '../../components/FormPasswordWithBalloons';
+import Checkbox from '../../components/Settings/Checkbox';
```

Comment:
I think we're either going to want to update the Settings checkbox to match content-server's (preferred by me at least), or make a new `Checkbox` component. I worked closely with UX on these checkbox states and I know they'd prefer we keep those that we have set in the `input-checkbox` class.